website/meta/universe.json

```diff
@@ -2604,7 +2638,7 @@
                 "    Add the courgette, garlic, red peppers and oregano and cook for 2–3 minutes.",
                 "    Later, add some oranges and chickens.\"\"\"",
                 "",
-                "# use any model that has internal spacy embeddings", 
+                "# use any model that has internal spacy embeddings",
```

Comment:
I notice you got rid of unnecessary spaces in a couple of places. That's OK but out of curiosity, is there a particular preprocessor you used that did this?

Reply:
I didn't not notice it my bad. I'm using the default PyCharm code formatter so I guess must have been that

Reply:
No worries, it's fine to include those changes, just wondered what caught it.